test/verify/check-pages

```diff
@@ -24,6 +24,21 @@ from testlib import *
 
 
 class TestPages(MachineCase):
+    def checkDocs(self, items):
+        m = self.machine
+        b = self.browser
+
+        b.click("#navbar-docs > a")
+        b.wait_visible("#navbar-docs-items")
+        expected = "Web Console"
+        expected += "".join(items)
+        # DOCUMENTATION_URL is only in Fedora
+        # RHEL is tracked in rhbz#1789984
+        if "fedora" in m.image:
+            expected = "Fedora documentation" + expected
```

Comment:
Curious that this works -- so `wait_collected_text()` ignores all space? Like this it will look for "documentationWeb Console", which is confusing. Could this get a space? Likewise in teh `"".join()` above?

Reply:
The method is already implemented like that and we use it on a few places (also in podman).
It literally just picks all text  from all matching elements. It does not add any spaces or anything if text does not contain it.